### ISSUE TITLE
Add DIMV correction (Sky Transmission) to pipeline

### ISSUE THREAD

**Issue by jburkepile** [ncar-mlso/kcor-pipeline#16]
DIMV keyword and value is in every KCOR fits header. Need to multiply data by ratio of DIMV of science image to DIMV value of flat image.

**Comment by mgalloy** [comment]
Put the flat DIMV value into the netCDF calibration file. Compare DIMV value to "yesterday's" (the value used to calibrate the day) value and act accordingly, e.g., notification.

**Comment by jburkepile** [comment]
The correction has been added and appears to be working correctly. The level1 science images are multiplied by the ratio of the SGSDIMV value of the flat to the SGSDIMV value of the science image.
We also added a keyword  'SKYTRANS' to the level 1 fits header that provides the value applied.

Mike processed 6 days in Nov 2017: 11/2, 11/5, 11/10, 11/12, 11/13, 11/21
I generated subtraction images to assess validity of the correction. The change in image brightness that was present due to the change in sky transmission has now been removed. There are times when the sky transmission changes abruptly. (11/10, 11/12, 11/13, 11/21). I carefully examined these periods. The quality control program rejected all images of abrupt significant sky transmission changes.
